mkosi/kmod.py

```diff
@@ -180,6 +181,15 @@ def gen_required_kernel_modules(
             mods = set(modulesd.rglob("*.ko*"))
         firmware = set()
 
+    # Some firmware dependencies are symbolic links, so the targets for those must be included in the list
+    # of required firmware files too. Links are not resolved recursively, and so links pointing to links
```

Comment:
"links are not resolved recursively" => "intermediate symlinks are not included, ..."